### PR TITLE
dev-python/seaborn: version bump 0.10.1

### PR DESCRIPTION
**Pull request by TheChymera**
Bug: https://bugs.gentoo.org/738374
Package-Manager: Portage-3.0.2, Repoman-2.3.23
Signed-off-by: Horea Christian <chr@chymera.eu>

### CODE REVIEW

**Comment by TheChymera** [comment]
@SoapGentoo why is my email address struck through in the gentoo bot comment above ( https://github.com/gentoo/gentoo/pull/17222#issuecomment-678719752 )?

**Comment by TheChymera** [comment]
@SoapGentoo ping?

**Comment by SoapGentoo** [comment]
@TheChymera maybe it's not the same one we have on file as maintainer email?

File: dev-python/seaborn/seaborn-0.10.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+inherit distutils-r1 virtualx
```
Comment (SoapGentoo) [inline dev-python/seaborn/seaborn-0.10.1.ebuild:8]: ... so you don't need `virtualx` or what?

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

File: dev-python/seaborn/seaborn-0.10.1.ebuild (hunk 1 above)
Comment (TheChymera) [reply dev-python/seaborn/seaborn-0.10.1.ebuild]: apparently it's no longer needed, yes

File: dev-python/seaborn/seaborn-0.10.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} )
```
Comment (SoapGentoo) [inline dev-python/seaborn/seaborn-0.10.1.ebuild:6]: any python 3.8 is gone again

File: dev-python/seaborn/seaborn-0.10.1.ebuild [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+inherit distutils-r1
+
+DESCRIPTION="Statistical data visualization"
+HOMEPAGE="https://seaborn.pydata.org https://github.com/mwaskom/seaborn"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+SLOT="0"
+LICENSE="BSD"
```
Comment (SoapGentoo) [inline dev-python/seaborn/seaborn-0.10.1.ebuild:15]: `LICENSE` comes before `SLOT`

File: dev-python/seaborn/seaborn-0.10.0.ebuild [hunk 4]
```diff
@@ -15,7 +15,7 @@ SLOT="0"
 LICENSE="BSD"
 KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
 IUSE="test"
-RESTRICT="!test? ( test )"
+RESTRICT="test"
```
Comment (SoapGentoo) [inline dev-python/seaborn/seaborn-0.10.0.ebuild:18]: please do this in a separate commit and not in the bump commit

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

File: dev-python/seaborn/seaborn-0.10.0.ebuild (hunk 4 above)
Comment (TheChymera) [reply dev-python/seaborn/seaborn-0.10.0.ebuild]: should I add the commit to this PR, or do another one after this is merged?

Comment (TheChymera) [reply dev-python/seaborn/seaborn-0.10.0.ebuild]: well, ok, I just have the version bump in this commit now.

**Comment by TheChymera** [comment]
@SoapGentoo ping?

**Pull request closed without merging**